### PR TITLE
Add blast

### PR DESCRIPTION
**Pull request by larrynats**

### CODE REVIEW

**Comment by larrynats** [comment]
Please review this, @casey-collab

File: english/blast/blast.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You put your hands together, charge up your energy, and shoot a powerful blast that extinguishes the marshmellow.
```
Comment (casey-collab) [inline english/blast/blast.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by larrynats** [comment]
Please review this, @casey-collab

File: english/blast/blast.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (casey-collab) [inline english/blast/blast.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
It looks like the file `english/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by larrynats** [comment]
Please review this, @casey-collab

File: english/blast/blast.md (hunk 2 above)
Comment (casey-collab) [inline english/blast/blast.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by larrynats** [comment]
Please review this, @casey-collab

File: english/blast/blast.md (hunk 2 above)
Comment (casey-collab) [inline english/blast/blast.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@larrynats, thanks for making revisions such as deleting the `.DS_Store` file! For the link, Casey is not requesting that your file contain a link (although adding one is certainly fine). Instead, the question is how a reader will get to your new page. Right now, if a reader starts by selecting their language in `language.md` and proceeds by clicking links, they can never reach your file no matter what link they click (unless I am mistaken). So you should add a link to another file that makes it possible for a reader to reach `blast.md`.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to address the comments above, feel free to re-open it.

**Pull request closed without merging**